src/components/chip-set/chip-set.tsx

```diff
@@ -401,10 +401,6 @@ export class ChipSet {
      * @returns {void}
      */
     private handleTextFieldFocus() {
-        if (this.editMode) {
-            return;
-        }
-
```

Comment:
This reintroduces the bug that was solved in #985

I had a look through the code, and it seems like we are not emitting `startEdit` when we explicitly set the focus through the `setFocus` method, maybe that will solve this? E.g.

```diff
    @Method()
    public async setFocus(emptyInput: boolean = false) {
        this.editMode = true;
        if (emptyInput) {
            this.textValue = '';
        }

        this.host.shadowRoot.querySelector('input').focus();
+       this.startEdit.emit();
    }
```